src/client/game.cpp

```diff
@@ -3245,12 +3240,10 @@ void Game::handlePointingAtNode(const PointedThing &pointed,
 						SimpleSoundSpec();
 
 				if (def.node_placement_prediction.empty() ||
-						nodedef_manager->get(map.getNode(nodepos)).rightclickable) {
+						nodedef_manager->get(map.getNode(nodepos)).can_place_onto)
```

Comment:
`rightclickable` is not the same as `can_place_onto`.

Reply:
Okay...